cep-XXXX.md

```diff
@@ -0,0 +1,424 @@
+| repodata_from_packages.json.gz | `repodata_from_packages.json.gz` file | application/vnd.conda.repodata_from_packages.v1+json+gzip |
+| repodata_from_packages.json.bz2 | `repodata_from_packages.json.bz2` file | application/vnd.conda.repodata_from_packages.v1+json+bz2 |
+| patch_instructions.json | `patch_instructions.json` file | application/vnd.conda.patch_instructions.v1+json |
+| patch_instructions.json.zst | `patch_instructions.json.zst` file | application/vnd.conda.patch_instructions.v1+json+zst |
+| patch_instructions.json.gz | `patch_instructions.json.gz` file | application/vnd.conda.patch_instructions.v1+json+gzip |
+| patch_instructions.json.bz2 | `patch_instructions.json.bz2` file | application/vnd.conda.patch_instructions.v1+json+bz2 |
+
+### OCI Artifact Structure for Conda Packages
+
+In order to store a conda package as an OCI artifact, we need to define how the data within the conda package is mapped
+into the OCI blobs and what additional metadata (if any) is stored in the OCI manifest.
+
+A conda package as an OCI artifact MUST have the following layers:
+
+- a layer holding either the `.tar.bz2` or `.conda` package file with the mediaType
+  `application/vnd.conda.package.v1` or `application/vnd.conda.package.v2`, respectively.
+- a layer holding the `info` folder as a gzipped tarball with extension `.tar.gz` and mediaType `application/vnd.conda.info.v1.tar+gzip`.
+- a layer holding the `info/index.json` file as plain JSON with the mediaType `application/vnd.conda.info.index.v1+json`.
+
+Additional layers are NOT allowed in the OCI artifact. If a conda package exists in both `.tar.bz2` and `.conda` format, the `.conda`
+version MUST be chosen to be in the OCI artifact.
+
+The manifest MUST have the following [OCI Annotations][annotations]
+
+- `org.conda.oci.schema` with the value `1` indicating the `v1` specification (defined by this document).
+- `org.conda.package.name` with the name of the conda package.
+- `org.conda.package.version` with the version of the conda package.
+- `org.conda.package.build` with the build string of the conda package
+
+Additional annotations under the `org.conda` namespace are NOT allowed.
+
+### OCI Artifact Structure for Repodata, etc
+
+All artifacts in the following OCI repositories
+
+```text
+<OCI conda channel base URL>/<OCI-compatible subdir>/repodata.json
+<OCI conda channel base URL>/<OCI-compatible subdir>/repodata_from_packages.json
+<OCI conda channel base URL>/<OCI-compatible subdir>/run_exports.json
+<OCI conda channel base URL>/<OCI-compatible subdir>/patch_instructions.json
+```
+
+MUST have a layer with the data as JSON and the appropriate mediaType from the table above. Additional layers with
+`gzip`, `bz2`, or `zst` compressed copies of the data MAY be included.
+
+The manifest MUST have the following [OCI Annotation][annotations]:
+
+- `org.conda.oci.schema` with the value `1` indicating the `v1` specification (defined by this document).
```

Comment:
Same here: shall we annotate channel, subdir and label in case they end up being hashed?

Reply:
They would not be hashed. See above.

Reply:
ah sorry. these entries are never hashed so no need here.